actors/builtin/miner/miner_actor.go

```diff
@@ -1621,11 +1621,18 @@ func processEarlyTerminations(rt Runtime) (more bool) {
 		unlockedBalance := st.GetUnlockedBalance(rt.CurrentBalance())
 		penaltyFromVesting, penaltyFromBalance, err := st.PenalizeFundsInPriorityOrder(store, rt.CurrEpoch(), penalty, unlockedBalance)
```

Comment:
We've had to do this in the past because we needed a way to track all of the funds we have already burned from our balance.  I'm going to do #1026 which should clean this up a lot but tracking all our penalties in FeeDebt without restriction.